gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.0.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.1.1-all.zip
```

Comment:
You should also regenerate the wrapper along with this change.
Eg. use `./gradlew wrapper` command

Reply:
Not necessary.

Reply:
Wrapper jar has changed between 7.0.2 and 7.1.1. See: https://gradle.org/release-checksums/
So necessary in case of those 2 versions.

Reply:
Actually the jar just have beed updated times, I don't know the reason why we must update it.

![image](https://user-images.githubusercontent.com/10363352/126994745-b0023f65-3560-4a23-b463-370c5d4a4cdb.png)

Reply:
https://docs.gradle.org/current/userguide/gradle_wrapper.html#sec:upgrading_wrapper

> Note that running the wrapper task once will update gradle-wrapper.properties only, but leave the wrapper itself in gradle-wrapper.jar untouched.

Reply:
OK, I executed `./gradlew wrapper`.